### DISCUSSION TITLE
Game crashed, has been happening every time I boot it up

### DISCUSSION THREAD

**Discussion by SmallAstronomyGremlin** [FabricMC/fabric-api#3274] [Player support]
### Description

Whenever I boot up my Minecraft with mods on a particular Instance that I made myself, this happens:

The game crashed whilst initializing game
Error: java.lang.RuntimeException: Mixin transformation of net.minecraft.class_329 failed
Exit Code: -1

I don't know what to do
![image](https://github.com/FabricMC/fabric/assets/89859134/df05d3ed-d6ef-491d-b469-a087704ef7b0)

**Comment by Poopooracoocoo** [comment]
Got a full report?
